apps/daimo-mobile/package.json

```diff
@@ -45,7 +45,7 @@
     "depcheck": "^1.4.3",
     "eslint": "^8.44.0",
     "eslint-config-universe": "^11.3.0",
-    "expo": "^49.0.7",
+    "expo": "*",
```

Comment:
intentional?

Reply:
happens automatically from `syncpack fix-mismatches`, should it not?

Reply:
~hmm running `npm run fix:deps` in project route doesn't do that for me~

**figured out why.** you'll have to add `@daimo/expo-passkeys` to .syncpackrc.js